static/_redirects

```diff
@@ -454,3 +454,4 @@
 /docs/concepts/overview/what-is-kubernetes/      /docs/concepts/overview/ 301
 /docs/tasks/administer-cluster/verify-signed-images/     /docs/tasks/administer-cluster/verify-signed-artifacts/   301
 
+/kubertenes     /blog/2024/06/06/10-years-of-kubernetes/ 
```

Comment:
Fixing word:

```suggestion
/kubernetes     /blog/2024/06/06/10-years-of-kubernetes/ 
```

Reply:
Apparently, this a “pun intended” since the blog for 10 years of k8s an so kuber**ten**es is intended. Thanks for clarifying in slack